classwork/class3/teddit.rb

```diff
@@ -2,7 +2,23 @@
 # This program will prompt the user for a headline, scan the headline for the word "cat" and the program will automatically assign fake upvotes to the story if there is the mention of a cat.
 
 
+
 # first, lets declare a method that will scan the story for the word cat or bacon. Let's name it calculate_upvotes
+<<<<<<< HEAD
```

Comment:
@brooklynzhu Got some merge conflicts that you are committing here. Can you grab the classwork on my repo, commit that in here, then push up?